tutorials/1_GettingREMIND.md

```diff
@@ -1,98 +1,54 @@
+- you can also edit your computer's advanced settings and add the GAMS path to the PATH variable manually (applies also if GAMS is installed but not included in PATH).
 
 This tutorial shows how to check and add variables to your PATH variable: <https://www.youtube.com/watch?v=5P9EDJwfXBo>
 
-Please add the GAMS training license you have been provided (gamslice.txt) by saving the file to your GAMS local folder. Under Windows something like `C:\Program Files (x86)\GAMS\28.2`
+Please add the GAMS training license you have been provided (gamslice.txt) by saving the file to your local GAMS folder. Under Windows something like `C:\Program Files (x86)\GAMS\<version number>`
 
-In addition *R* (<https://www.r-project.org/>) is required for pre- and postprocessing and run management (needs to be added to the user's PATH variable as well). It is recommended to install also RSudio (<https://www.rstudio.com>).
+In addition *R* (<https://www.r-project.org/>) is required for pre- and postprocessing and run management (needs to be added to the user's PATH variable as well). It is recommended to also install RSudio (<https://www.rstudio.com>).
 
-For R, some packages are required to run REMIND. All are either distributed via the offical R CRAN or via a separate repository hosted at PIK (PIK-CRAN). Before proceeding PIK-CRAN should be added to the list of available repositories via:
+On Windows you need to install [Rtools](https://cran.r-project.org/bin/windows/Rtools/) and add it to the system PATH variable.
 
-``` r
-options(repos = c(CRAN = "@CRAN@", pik = "https://rse.pik-potsdam.de/r/packages"))
-```
+For some types of REMIND output post-processing, LaTeX (<https://www.latex-project.org/get/>) and pandoc (<https://pandoc.org/installing.html>) are required. Make sure the executable "pdflatex" is added to the PATH variable of your system.
 
-On Windows you need to install [Rtools](https://cran.r-project.org/bin/windows/Rtools/) and add it to the system PATH
-variable. After that all required packages can be installed via `install.packages`:
+Navigate to the REMIND folder that you cloned earlier. Always start R scripts and sessions related to REMIND from this path. To install all required R packages and check if your system is ready to run REMIND execute the following:
 
-``` r
-pkgs <- c(
-  "colorspace",
-  "crayon",
-  "data.table",
-  "devtools",
-  "dplyr",
-  "edgeTrpLib",
-  "flexdashboard",
-  "gdx",
-  "gdxdt",
-  "gdxrrw",
-  "ggplot2",
-  "gms",
-  "gridExtra",
-  "gtools",
-  "htmltools",
-  "knitr",
-  "lazyeval",
-  "lucode2",
-  "luplot",
-  "luscale",
-  "lusweave",
-  "magclass",
-  "magpie4",
-  "mip",
-  "mrremind",
-  "mrvalidation",
-  "optparse",
-  "plotly",
-  "quitte",
-  "remind2",
-  "reshape2",
-  "rlang",
-  "rmarkdown",
-  "rmndt",
-  "stringr",
-  "tidyr",
-  "tidyverse",
-  "withr",
-  "yaml"
-)
-install.packages(pkgs)
+```sh
+Rscript checkSetup.R
 ```
```

Comment:
> Navigate to the REMIND folder that you cloned earlier. Always start R scripts and sessions related to REMIND from this path. To install all required R packages and check if your system is ready to run REMIND execute the following:
> 
> ```sh
> Rscript checkSetup.R
> ```

This does not work for new users.  There is no mention anywhere of the RSE repository and how to set it.  The local `.Rprofile` would set it only after hydration, which will have failed by then.  Moving that up (see above) resolves this.